tasks/connectors/qualys_was/qualys_was.rb

```diff
@@ -123,6 +128,7 @@ def run(opts)
                   max_records = 0
                 end
                 find_from = data["Finding"]
+                abort("Error: No finding data found. Exiting the process") if find_from.nil?
```

Comment:
We wanted to avoid nil vuln_def name because importing that would cause an entire connector run to fail. By `abort`-ing here we are still failing an entire toolkit task run if there is one bad finding. We want to avoid that. I would _only_ do `print_error()` when we are missing this data.

Also if we want to stop execution, there's a `fail_task` helper method we can use: https://github.com/KennaSecurity/toolkit/blob/main/lib/helpers.rb#L77

Reply:
Got it! I will refactor the code :) Thank you